packages/binding-http/test/http-client-oauth-tests.ts

```diff
@@ -22,41 +22,43 @@ import { OAuth2SecurityScheme } from "@node-wot/td-tools";
 import InMemoryModel from "./memory-model";
 import { promisify } from "util";
 import { ProtocolHelpers } from "@node-wot/core";
-const OAuthServer = require("express-oauth-server");
-const bodyParser = require("body-parser");
+import { readFileSync } from "fs";
 
-const fs = require("fs");
+import OAuthServer from "express-oauth-server";
+import bodyParser from "body-parser";
+
+import { AuthorizationCodeModel } from "oauth2-server";
 
 @suite("HTTP oauth client implementation")
 class HttpClientOAuthTest {
     private client: HttpClient;
-    static model: any;
+    static model: AuthorizationCodeModel;
```

Comment:
```suggestion
    static model: InMemoryModell;
```

Should solve also the other `any`s, have you tried it?

Reply:
I think I tried it out before, as setting the type of `model` to `InMemoryModel` leads to the following errors 😕

```
Type 'InMemoryModel' is not assignable to type 'AuthorizationCodeModel | ClientCredentialsModel | RefreshTokenModel | PasswordModel | ExtensionModel'.
  Property 'verifyScope' is missing in type 'InMemoryModel' but required in type 'RefreshTokenModel'.ts(2322)
index.d.ts(285, 9): 'verifyScope' is declared here.
index.d.ts(170, 9): The expected type comes from property 'model' which is declared here on type 'Options'
```

I think `InMemoryModel` should probably implement/extend one of the mentioned types so that we can get rid of the type casting.